### ISSUE TITLE
Pandoc 2.7.3 hangs on Windows 10. Pandoc 2.7.2 unaffected.

### ISSUE THREAD

**Issue by ikehoff** [jgm/pandoc#5646]
I've installed Pandoc via Chocolatey in the past and have continued to upgrade via Choco to version 2.7.3. This version appears to be broken.

System information:
Microsoft Windows 10 Pro
10.0.18362 Build 18362
Surface Book 2
x64-based PC

I noticed 2.7.3 will hang on any command. Basic commands I've tried are "pandoc -v" and "pandoc -h"; neither produce any output and will hang at a blank newline (does not return to the prompt).

Forcing the install back to pandoc 2.7.2 resolves this issue. The command issued is: cinst pandoc --version 2.7.2 --force -y

I have not performed any further diagnosis but can assist when I have more time to dedicate to this issue.

**Comment by jgm** [comment]
Can you try the Windows binary we provide (under Releases) instead of the chocolatey one?  (You can download a zip and just run the executable directly, to make sure you're using that one.)

We need to determine whether this is a chocolatey issue or a pandoc issue. Since no other Windows users have complained about this, my guess is the former (in which case you'd need to report it to them, not here).

**Comment by ikehoff** [comment]
The Windows binary from the 2.7.3 release on GitHub works fine. (zip file)

The Windows installer from the 2.7.3 release on GitHub works fine for single user and allusers. (msi file)

Not sure why the chocolatey install would be any different. It appears the URL for the msi is the same.

**Comment by ikehoff** [comment]
Alright, got it working. One thing I noticed is the Pandoc install package is NOT performing any integrity checking. Seems there is some orphaned code included.

I also discovered what appears to be a necessary powershell module. The module "au" is being imported but nothing is installing it first so the import silently fails. It looks like package dependencies are broken or not included at all. 

The fix (I think):
Installed the AU package with 'cinst au -y'
Installed pandoc with the allusers flag set 'cinst pandoc -y -ia "ALLUSERS=1"'
Tested pandoc with 'pandoc -v' and it works.
